sqs/main.tf

```diff
@@ -65,12 +65,11 @@ resource "aws_sqs_queue" "dlq_with_sse" {
 }
 
 module "dlq_cloudwatch_alarm" {
-  count               = var.create_cloudwatch_alarm ? 1 : 0
   source              = "../cloudwatch_alarms"
   metric_name         = "ApproximateNumberOfMessagesVisible"
   namespace           = "AWS/SQS"
   name                = "${var.queue_name}-dlq-alarm"
```

Comment:
Worth adding something like "messages-visible" as the
 DLQ could have other alarms?

Reply:
Good point, I've added it in.

Reply:
I think the DLQ will also need this.

Reply:
Fair enough, I've changed this one as well.